packages/components/src/components/tabs-bar/tabs-bar.tsx

```diff
@@ -247,12 +248,13 @@ export class TabsBar {
       // tab the first slotted one with tabindex=0 becomes focused instead of the one after,
       // therefor the 'Tab' case needs to be handled
       case 'Tab':
-        const { target } = e;
-        const { tabIndex } = target;
-        target.tabIndex = null;
-        setTimeout(() => {
-          target.tabIndex = tabIndex;
-        });
+        if (target.matches('button')) {
+          const { tabIndex } = target;
+          target.tabIndex = null;
+          setTimeout(() => {
+            target.tabIndex = tabIndex;
+          });
```

Comment:
What does this do? Is it essentially just reassigning the same tabIndex value again? Maybe add a comment

Reply:
It's not a new feature. And there is alre4ady a comment 😉.